sim-cli/src/main.rs

```diff
@@ -14,19 +14,37 @@ use simple_logger::SimpleLogger;
 struct Cli {
     #[clap(long, short)]
     config: PathBuf,
+    #[clap(long, short)]
+    total_time: Option<u32>,
+    #[clap(long, short)]
+    debug: bool,
 }
 
 #[tokio::main]
 async fn main() -> anyhow::Result<()> {
+    let cli = Cli::parse();
```

Comment:
would we also want to read in `RUST_LOG` environment at this point?

Reply:
I don't think I follow